iocage_lib/ioc_json.py

```diff
@@ -2415,6 +2436,17 @@ def json_check_prop(self, key, value, conf, default=False):
                             silent=self.silent,
                             exception=ioc_exceptions.ValidationFailed
                         )
+                elif key == 'mac_prefix':
+                    if not self.validate_mac_prefix(value):
+                        iocage_lib.ioc_common.logit(
+                            {
+                                'level': 'EXCEPTION',
+                                'message': 'Please specify a valid mac_prefix'
+                                           'which has the first bit in the'
+                                           'first byte unset and the second'
+                                           'bit set.'
```

Comment:
This is Greek for 99% of the users :P 
Can we simplify and say mac address cannot start with "XX" or "YY" ?

Reply:
How about:

```
Invalid mac_prefix. Must match `?X` where ? can be any valid hex digit (0-9, A-F) and X is one of 2, 6, A or E.
```